pkg/aggregator/aggregator_test.go

```diff
@@ -499,31 +509,34 @@ func TestTags(t *testing.T) {
 	}
 }
 
-func TestAggregatorFlush(t *testing.T) {
+func TestTimeSamplerFlush(t *testing.T) {
 	defer config.Datadog.Set("aggregator_flush_metrics_and_serialize_in_parallel", nil)
 
 	tests := []struct {
 		name    string
 		enabled bool
 	}{
-		{
-			name:    "aggregator_flush_metrics_and_serialize_in_parallel false",
-			enabled: false,
-		},
-		{
-			name:    "aggregator_flush_metrics_and_serialize_in_parallel true",
-			enabled: true,
-		},
+		{name: "aggregator_flush_metrics_and_serialize_in_parallel false", enabled: false},
+		{name: "aggregator_flush_metrics_and_serialize_in_parallel true", enabled: true},
 	}
 
+	pc := config.Datadog.GetInt("dogstatsd_pipeline_count")
+	config.Datadog.Set("dogstatsd_pipeline_count", 1)
+	defer func() {
```

Comment:
Just because sometimes, doing this in unit tests, I have to do it for multiple configuration fields, but you are right that I could not have used a lambda here that would make no difference.